kw/platform/settings.py

```diff
@@ -22,3 +23,9 @@
 
 #: Status message sent in response to requests with invalid `User-Agent`.
 KIWI_RESTRICT_USER_AGENT_MESSAGE = "Invalid User-Agent: does not comply with KW-RFC-22"
+
+#: Enable inspection of requests' User-Agent header and their restriction if not
+#: compliant with ``KW-RFC-22``, ``True`` by default.
+KIWI_ENABLE_RESTRICTION_OF_REQUESTS = json.loads(
```

Comment:
Hmm, this will restrict it to `true` and `false` values, which isn't bad, but it's not how usually this values are set.
Using `distutils.util.strtobool` instead of `json.loads` seems more flexible, what do you think?

Reply:
Yep, definitely much better :-)